robottelo/api/utils.py

```diff
@@ -814,6 +814,17 @@ def update_vm_host_location(vm_client, location_id):
     host.update(['location'])
 
 
+def update_smart_proxy_location(smart_proxy, location_id):
```

Comment:
I feel that helper is absolutely redundant as such things should be used inline, directly in the code. It is not the same as method `update_vm_host_location` above. Imagine if we add helper per every entity and per each entity attribute.

Reply:
If you would explain this to me in your first attempt of pr review, it would have saved some amount of my time. I hope you understand this :)

Reply:
haha, man, I completely understand that, but it is really hard to have proper focus with such amount of PRs that I reviewed and reviewing. I feel your pain, but it was not intentional and never will be to waste your time for nothing

Reply:
Only one recommendation, just re-run tests once you feel that we finish with review. In that case, impact on your time will be minimal